src/main/java/uk/gov/companieshouse/documentsigningrequestconsumer/satisfyitem/SatisfyItemApiPatch.java

```diff
@@ -48,4 +50,16 @@ public void satisfyItem(ServiceParameters parameters, int status, String documen
         logger.info("API returned response: "+ response.getStatusCode(),
             getLogMap(parameters.getData().get(ORDER_ID).toString(), parameters.getData().get(GROUP_ITEM).toString()));
     }
+
+    private String getGroupItemExcludingItemId (final String groupItem)
+    {
+        String groupItemExcludingItemId = "";
+        int lastIndex = 0;
+
+        if ((lastIndex = groupItem.lastIndexOf('/')) != -1) {
```

Comment:
So the `itemGroupsUri` becomes an empty string if it previously contained no `/` character? This may be an unlikely scenario, but if we're saying that correct behaviour depends on there being at least one `/` in the string to start with, then maybe it's better to error clearly as soon as we know there's something wrong with our input?